index.html

```diff
@@ -2,33 +2,143 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="/styles/styles.css">
     <title>Concept-Opdracht</title>
 </head>
 <body>
-<header>
-    <div>
-        <nav>
+<header class="outer-content-container-header">
+    <div class="inner-content-container-header">
+        <nav class="nav-bar">
             <ul>
                 <li><a href="#case-studies"></a>Case studies</li>
                 <li><a href="#services"></a>Services</li>
                 <li><a href="#contact"></a>Contact us</li>
             </ul>
         </nav>
-    </div>
 
-    <div class="header-text">
-    <h1> Your website is an investment in the success of your business.<h1/>
-        <p class="to-help">We are here to help you succeed.<p/>
+    <div class="header-text" >
+        <h1> Your website is an investment in the success of your business.</h1>
+        <p class="to-help">We are here to help you succeed.
+        </p>
         <button type="button" class="contact-button">Let's work together</button>
     </div>
+    </div>
 
-        <div class="squiggle-stripe">
-
+    <div class="squiggle-stripe">
     </div>
 
-    </header>
+</header>
+
+<main>
+    <section class="outer-content-container-introduction">
+        <div id="introduction" class="inner-content-container-introduction">
+            <h3>
+                Rekupe is a digital agency in Los Angeles.
+            </h3>
+            <h3>
+                We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.
+            </h3>
+            <h3>
+                We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+            </h3>
+            <p class="intro_link">Have a question? <a href="https://"> Click here</a> to contact us!</p>
+
+        </div>
+    </section>
+
+    <section id="projects">
+        <article class="inner-content-container-media" id="rolling-stones-project">
+            <div class="tekst-rolling-stones">
+                <h3>Rolling Stone</h3>
+                <div class="squiggle_line1"></div>
+                <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global
+                    icon.</p>
+                <p> <a href="https://" class="view-project"> view project</a> </p>
+            </div>
+            <span class="image-wrap-rolling-stones">
+                <img id="image-wrap-rolling-stones-wrap" src="../Frontend-html-css-concept-opdracht/assets/work-image-01.jpeg" alt="project rolling stones">
```

Comment:
Alle paden van de afbeeldingen kloppen niet
```suggestion
                <img id="image-wrap-rolling-stones-wrap" src="./assets/work-image-01.jpeg" alt="project rolling stones">
```